client/src/io/rapidpro/androidchannel/SyncService.java

```diff
@@ -405,14 +405,6 @@ protected void doWakefulWork (Intent intent) {
             return;
         }
 
-        // if our endpoint is an ip, add :8000 to it
-        if (endpoint.startsWith("ip")){
```

Comment:
Though I'm ok removing the 8000 port default, you'll still need the rest of this block.